src/components/MenuItem/MenuItem.tsx

```diff
@@ -1,28 +1,21 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
```

Comment:
I just noticed that the styling of disabled menuItem on the dropdown is rounded now.

![Screenshot 2022-11-24 at 10 57 21](https://user-images.githubusercontent.com/35405438/203755397-a72ddb73-18eb-479a-84fa-fa7648550769.png)

Reply:
Fixed. Idk why I added a `tw-rounded` class to the `MenuItem` container. Probably on the first tests and I forgot to remove 🤦‍♂️ Thank you so much for catching this 💪